### PR TITLE
Add localhostProfile attribute and column

### PR DESCRIPTION
**Pull request by cmurphy**
Add a new attribute, `localhostProfile`, to the SeccompProfile CRD and
ensure it is exposed via kubectl's `-o wide` formatting option. This
attribute represents the string that a user would have to supply to
their container, pod, or application (Deployment, ReplicaSet, DaemonSet,
etc) manifest in order for their application to use the profile. This
makes it clearer than just checking the Path attribute, since it's not
obvious that the `/var/lib/kubelet/seccomp/` part of the path is not
supposed to be included in the pod spec.

#### What type of PR is this?

/kind feature
/kind api-change
/kind documentation

#### What this PR does / why we need it:

This is what I have in mind for https://github.com/kubernetes-sigs/security-profiles-operator/issues/145#issuecomment-740063350 - what do you think?

#### Which issue(s) this PR fixes:

Relates to #145 

#### Does this PR introduce a user-facing change?

```release-note
Adds a new attribute `status.seccompProfile\.localhostProfile` and column `SECCOMPPROFILE.LOCALHOSTPROFILE` to indicate what should be included in a pod spec.
```

### CODE REVIEW

File: api/v1alpha1/seccompprofile_types.go [hunk 1]
```diff
@@ -96,6 +96,8 @@ type SeccompProfileStatus struct {
 	Path            string   `json:"path,omitempty"`
 	Status          string   `json:"status,omitempty"`
 	ActiveWorkloads []string `json:"activeWorkloads,omitempty"`
+	// The path the profile to provide in a pod spec
+	SeccompProfileLocalhostProfile string `json:"seccompProfile.localhostProfile,omitempty"`
```
Comment (saschagrunert) [inline api/v1alpha1/seccompprofile_types.go:100]: What about:
```suggestion
	LocalhostProfile string `json:"localhostProfile,omitempty"`
```
And then we enhance the description to mention that is is the container or pod `securityContext.seccompProfile.localhostProfile` path.

File: installation-usage.md [hunk 2]
```diff
@@ -81,6 +81,36 @@ spec:
       image: nginx
 ```
 
+You can find the profile path of the seccomp profile by checking the
+`seccompProfile.localhostProfile` attribute:
+
+```sh
+$ kubectl --namespace my-namespace get seccompprofile profile1 --output wide
+NAME       STATUS   AGE   SECCOMPPROFILE.LOCALHOSTPROFILE
+profile1   Active   14s   operator/my-namespace/custom-profiles/profile1.json
```
Comment (saschagrunert) [inline installation-usage.md:90]: I really like that, that will improve the usability a lot!

**Review by saschagrunert — COMMENTED** [review]
Generally LGTM, just one thought on the naming of the field :point_down:

File: api/v1alpha1/seccompprofile_types.go (hunk 1 above)
Comment (cmurphy) [reply api/v1alpha1/seccompprofile_types.go]: done

**Review by saschagrunert — APPROVED** [review]
/lgtm

File: installation-usage.md (hunk 2 above)
Comment (pjbgf) [reply installation-usage.md:90]: That looks very neat, really good stuff @cmurphy. :+1:

**Pull request merged**